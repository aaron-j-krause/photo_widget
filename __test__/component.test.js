import React, { Component } from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'

const Comp = ({ xt }) => (
  <h1>{xt}</h1>
)

test('componnnennt', () => {
  let wrapper = shallow(<Comp txt='idk'/>)
  expect(wrapper.find('h1').text()).toBe('idk')
})

test('other commmponent', () => {
  let t = renderer.create(
    <Comp txt="boo" /> 
  )

  expect(t).toMatchSnapshot()
})
