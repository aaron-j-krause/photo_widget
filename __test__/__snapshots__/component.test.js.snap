exports[`test other commmponent 1`] = `
<h1>
  boo
</h1>
`;
